src/components/notifications/ConfigurationTab.tsx

```diff
@@ -202,6 +245,89 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
             )
     }
 
+    editWebhookHandler(e) {
+        let _state = {...this.state}
+        _state.showWebhookConfigModal = true;
```

Comment:
no need to create variables. we can directly pass to the setState

Reply:
done